taxi/forms.py

```diff
@@ -0,0 +1,40 @@
+from django import forms
+from django.contrib.auth.forms import UserCreationForm
+from django.core.exceptions import ValidationError
+from taxi.models import Driver
+
+
+def check_license_number(number: str):
```

Comment:
it is better to check the license to be precisely 8 symbols long here with a descriptive error message to keep everything in one place